eventarc/audit_iam/main_test.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	cloudevent "github.com/cloudevents/sdk-go/v2"
+	"github.com/googleapis/google-cloudevents-go/cloud/auditdata"
+	"github.com/googleapis/google-cloudevents-go/third_party/googleapis/google/api"
+)
+
+func TestIamAuditlogs(t *testing.T) {
```

Comment:
I'm definitely going to bookmark this test--I always get stuck whenever I need to write a test that records an HTTP response.